Makefile

```diff
@@ -0,0 +1,99 @@
+include Makefile-utils.mk
+
+#
+# Calypso
+#
+.PHONY: pre-start
+pre-start: welcom versions
```

Comment:
I assume you'll catch this but this should be `welcome` 😉

Reply:
thanks! fixed in a22a1dfc91 - my <kbd>e<kbd> key fails me more often than not anymore